README.md

```diff
@@ -1,2 +1,43 @@
-# rfcs
-The master record of all RFCs in development
+# Tessel RFCs
+
+For feature additions to the Tessel platform that require some aspect of design or discussion, an RFC (Request For Comment) is necessary. This helps the community collaborate to build robust, thoughtful, and effective additions to the platform. Our hope is to keep the process as light as possible while preserving the reliability and forward-looking nature of Tessel.
+
+Some things don't require RFCs! These are things like documentation fixes or additions or simple bug fixes. If you're not sure, open an issue and ask or pose the question on [the Tessel Slack channel](tessel-slack.herokuapp.com).
+
+## Active RFCs
+None yet! Read about the process below and be the first.
+
+## How to create an RFC
+Our RFC process is heavily inspired by the ECMA/TC39 RFC process. The process is as follows:
+
+### 1. Create a repo for the feature
+One or more champions propose a new feature by creating a repo dedicated to that feature
+Example: https://github.com/rwaldron/exponentiation-operator
+
+The feature is free to describe all aspects of the platform that it interacts with and is described as thoroughly as possible in the readme.md of the repo. Using a repo is convenient for adding and removing fellow feature contributors.
+
+### 2. Post about it on the Forums
+We strongly encourage new RFCs to be posted on [the Forums](forums.tessel.io) for the sake of including as many people as possible.
```

Comment:
I'm confused. Where do people go if they want to see a centralized list of all open RFCs?